src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/InitialConsiderationHelperTest.java

```diff
@@ -0,0 +1,154 @@
+import uk.gov.hmcts.et.common.model.ccd.items.DateListedTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.JurCodesTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.DateListedType;
+import uk.gov.hmcts.et.common.model.ccd.types.JurCodesType;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class InitialConsiderationHelperTest {
+
+    private static final String EXPECTED_RESPONDENT_NAME =
+            "| Respondent name given | |\r\n"
+                    + "|-------------|:------------|\r\n"
+                    + "|In Et1 by claimant | Test Corp|\r\n"
+                    + "|In Et3 by Respondent | |";
+
+    private static final String EXPECTED_RESPONDENT_NAME_BLANK =
+            "| Respondent name given | |\r\n"
+                    + "|-------------|:------------|\r\n"
+                    + "|In Et1 by claimant | |\r\n"
+                    + "|In Et3 by Respondent | |";
+
+    private static final String EXPECTED_HEARING_STRING =
+            "|Hearing Details | |\r\n"
+                    + "|-------------|:------------|\r\n"
+                    + "|Date | 01 Jul 2022|\r\n"
+                    + "|Type | Preliminary Hearing(CM)|\r\n"
+                    + "|Duration | 1.5 Hours|";
+
+    private static final String EXPECTED_HEARING_BLANK =
+            "|Hearing Details | |\r\n"
+                    + "|-------------|:------------|\r\n"
+                    + "|Date | |\r\n"
+                    + "|Type | |\r\n"
+                    + "|Duration | |";
+
+    private static final String EXPECTED_JURISDICTION_HTML = "<h2>Jurisdiction Codes</h2><a target=\"_blank\" " +
+            "href=\"https://intranet.justice.gov.uk/documents/2017/11/jurisdiction-list.pdf\">View all jurisdiction codes and descriptors (opens in new tab)</a><br><br>" +
+            "<strong>DAG</strong> - Discrimination, including indirect discrimination, harassment or victimisation or discrimination based on association or perception on " +
+            "grounds of age<br><br><strong>SXD</strong> - Discrimination, including indirect discrimination, discrimination based on association or perception, harassment " +
+            "or victimisation on grounds of sex, marriage and civil partnership or gender reassignment<br><br><hr>";
+
+    private CaseData caseDetailsEmpty;
+    private CaseData caseDetails;
+
+    @BeforeEach
+    public void setUp() throws Exception {
+        caseDetails = generateCaseData("initialConsiderationCase1.json");
+        caseDetailsEmpty = generateCaseData("initialConsiderationCase2.json");
+    }
+
+    @Test
+    public void getEarliestHearingDate() {
+        assertEquals(Optional.empty(), InitialConsiderationHelper.getEarliestHearingDate(new ArrayList<>()));

```

Comment:
Prefer Hamcrest as it makes for easier reading.  E.g.
```
assertThat(InitialConsiderationHelper.getEarliestHearingDate(new ArrayList<>()))
.isEmpty();
```

Also worth noting how we show this on 2 lines as that's like adding a comma in a sentence which just adds to the readability.
Worth applying across the board

Reply:
this is assert j syntax. Have used assert j